### ISSUE TITLE
Add Egit as default included feature

### ISSUE THREAD

**Issue by hqnghi88** [gama-platform/gama#2977]
Working on some (4) gaml project and all of them are on github. I would like to have the git gui (egit) installed as default in gama release, as it help to sync pull push directly in gama.

**Comment by sriramab** [comment]
I think this was my request long time ago, but the slowly I started feeling may be including a terminal inside GAMA is more useful, because it then allows to do many things: GIT, R, Python, Julia, Matlab.. because it is the system terminal but now inside GAMA

**Comment by sriramab** [comment]
For egit, i first install eclipse market place, and then you can install any plugin i want.
https://www.youtube.com/watch?v=fvNcYwA9Kxw

**Comment by AlexisDrogoul** [comment]
As stated in #3041 , 1.8.2 supports the installation of EGit, so I'm not sure if it is really necessary. An "Egit" GAMA plugin would make sense if we can provide additional features (a new virtual folder dedicated to remote models ? a better integration of the wizards ?...). What do you think ?

**Comment by hqnghi88** [comment]
I agree with the extended plugin where we could integrate more remote services for models (git, gdrive....) . The problem for me (or some who which version too much) that the metadata of git can cause problem (some error message but no critical error for the moment) when a gama version dont have git by default , not because it take time to install or the incompatibility  in 3041.
